frontend/src/components/VideoStream/VideoStreamWindow.tsx

```diff
@@ -1,50 +1,53 @@
-import { Button, Card, Icon, Typography } from '@equinor/eds-core-react'
+import { Card, Icon, Typography } from '@equinor/eds-core-react'
 import { tokens } from '@equinor/eds-tokens'
 import { fullscreen } from '@equinor/eds-icons'
 import styled from 'styled-components'
-import { VideoStream } from 'models/VideoStream'
 import { useState } from 'react'
+import ReactModal from 'react-modal'
+
+import { VideoStream } from 'models/VideoStream'
+import { VideoSection } from './VideoSection'
 
-interface VideoStreamProps {
+interface VideoStreamWindowProps {
     videoStream: VideoStream
 }
-
 const VideoCard = styled(Card)`
-    width: 400px;
     padding: 16px;
 `
 
-const VideoSection = styled(Card)`
-    display: inline-flex;
-`
-
-const FullscreenButton = styled(Button)`
+const VideoFullScreen = styled(ReactModal)`
     position: absolute;
-    position: absolute;
-    bottom: 8px;
-    right: 8px;
-    width: 32px;
-    height: 32px;
-    opacity: 0.75;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
 `
 
 Icon.add({ fullscreen })
 
-export function VideoStreamWindow({ videoStream }: VideoStreamProps) {
+export function VideoStreamWindow({ videoStream }: VideoStreamWindowProps) {
     const [fullScreenMode, setFullScreenMode] = useState<boolean>(false)
+
     const toggleFullScreenMode = () => {
         setFullScreenMode(!fullScreenMode)
     }
 
     return (
-        <VideoCard variant="default" style={{ boxShadow: tokens.elevation.sticky }}>
-            <VideoSection>
-                <img src={videoStream.url} alt="robot video stream" />
-                <FullscreenButton color="secondary" onClick={toggleFullScreenMode}>
-                    <Icon name="fullscreen" size={32} />
-                </FullscreenButton>
-            </VideoSection>
-            <Typography variant="h5">{videoStream.name}</Typography>
-        </VideoCard>
+        <>
+            <VideoCard variant="default" style={{ boxShadow: tokens.elevation.sticky }}>
+                <VideoSection videoStream={videoStream} toggleFullScreenModeFunction={toggleFullScreenMode} />
+                <Typography variant="h5">{videoStream.name}</Typography>
+            </VideoCard>
+            <VideoFullScreen
+                isOpen={fullScreenMode}
+                onRequestClose={() => {
+                    setFullScreenMode(false)
+                }}
+            >
+                <VideoCard variant="default" style={{ boxShadow: tokens.elevation.sticky }}>
+                    <Typography variant="h5">{videoStream.name}</Typography>
+                    <VideoSection videoStream={videoStream} toggleFullScreenModeFunction={toggleFullScreenMode} />
+                </VideoCard>
+            </VideoFullScreen>
```

Comment:
Won't this display both video cards (although one will be on top) since there is no check for the `fullScreenMode` on the normal video card?

Reply:
That is true, decided to have all video streams as small in the background as described in the UX design: 
![image](https://user-images.githubusercontent.com/32652118/183595343-5bbb1a73-828a-402c-86e9-a058aa16e7a6.png)
Should it be much to handle all the video streams at once we could disable the ones not in full screen mode, but i propose that we add this if it proves to be an issue later

Reply:
Sounds good!